integration-tests/tests/mocked/get-question/step-functions-local.test.ts

```diff
@@ -0,0 +1,107 @@
+import { HistoryEvent } from "@aws-sdk/client-sfn";
+import { SfnContainerHelper } from "./sfn-container-helper";
+
+jest.setTimeout(30_000);
+
+describe("step-function-local", () => {
+  describe("HMRC KBV Get Questions", () => {
+    let stnContainerHelper: SfnContainerHelper;
+    describe("is successful", () => {
+      beforeAll(async () => {
+        stnContainerHelper = new SfnContainerHelper();
+      });
+      afterAll(async () => stnContainerHelper.shutDown());
+      it("has a container is running", async () => {
+        expect(stnContainerHelper.getContainer()).toBeDefined();
+      });
+
+      describe("happy Case Scenario", () => {
+        it("should get 200 when session id present", async () => {
```

Comment:
This is testing more than just the `sessionId` being present. Currently, this is your only happy path test testing the entire flow, so can we rename this to be more accurate to that?

Reply:
Missing the HTTP 204 happy path test